src/commands/ext-dev-register.ts

```diff
@@ -52,8 +52,7 @@ export default new Command("ext:dev:register")
       throw new FirebaseError(
         `Failed to register publisher ID ${clc.bold(publisherId)} for project ${clc.bold(
           projectId
-        )}: ${err.message}`,
-        { exit: 1 }
```

Comment:
Is removing the exit code intentional?

Reply:
Yes - sine I was cleaning up the block, I removed some `exit: 1`s that I saw - that's the default value and doesn't need to be provided.